experimental/traffic-portal/src/app/core/cache-groups/regions/table/regions-table.component.ts

```diff
@@ -82,10 +82,9 @@ export class RegionsTableComponent implements OnInit {
 	];
 
 	/** Definitions for the context menu items (which act on augmented region data). */
-	public contextMenuItems: Array<ContextMenuItem<Region>> = [
+	public contextMenuItems: Array<ContextMenuItem<ResponseRegion>> = [
 		{
-			action: "edit",
-			multiRow: false,
+			href: (selectedRow: ResponseRegion): string => `/core/region/${selectedRow.id}`,
```

Comment:
This path doesn't match any routes. It's `/core/regions/:id`, with an `s`. That's my fault, I broke that by changing the route (which fixed the previously broken link on the "+" FAB)